### ISSUE TITLE
CRA (js and ts) webpack build failure

### ISSUE THREAD

**Issue by ioanabrooks** [aws-amplify/amplify-ui#3911]
### Before creating a new issue, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-ui/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-ui/discussions).
- [X] I have tried disabling all browser extensions or using a different browser
- [X] I have tried deleting the node_modules folder and reinstalling my dependencies
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md#bug-reports).

### On which framework/platform are you having an issue?

React

### Which UI component?

Authenticator, Account Settings, Geo (MapView), In-App Messaging, Primitive components, Storage (File Uploader)

### How is your app built?

CRA, CRA-ts

### Please describe your bug.

Webpack version 5.83.0 released this morning is throwing errors while building:

```
Creating an optimized production build...
Failed to compile.

HookWebpackError: Cannot read properties of undefined (reading 'e')
-- inner error --
TypeError: Cannot read properties of undefined (reading 'e')
Generated code for /home/runner/work/amplify-ui/amplify-ui/canary/apps/react/cra/node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[1].oneOf[5].use[1]!/home/runner/work/amplify-ui/amplify-ui/canary/apps/react/cra/node_modules/postcss-loader/dist/cjs.js??ruleSet[1].rules[1].oneOf[5].use[2]!/home/runner/work/amplify-ui/amplify-ui/canary/apps/react/cra/node_modules/source-map-loader/dist/cjs.js!/home/runner/work/amplify-ui/amplify-ui/canary/apps/react/cra/node_modules/@aws-amplify/ui-react/dist/styles.css
```

The workaround is to pin webpack to the previous version `5.82.1` .

Using yarn:
```
"resolutions": {
    "webpack": "5.82.1"
  }
```

Using npm:
```
"overrides": {
     "webpack": "5.82.1"
   }
```

### What's the expected behaviour?

Build should not fail.

### Help us reproduce the bug!

Create a new project using CRA or CRA-ts, and build using yarn or npm.

### Code Snippet

```typescript
// Put your code below this line.

```

**Comment by ioanabrooks** [comment]
Related CRA issue: https://github.com/facebook/create-react-app/issues/13171

**Comment by ioanabrooks** [comment]
Related webpack issue: https://github.com/webpack/webpack/issues/17213

**Comment by TheLarkInn** [comment]
It is now fixed in a patch update (5.83.1) and should be resolved as long as you don't have the version pinned :-)

**Comment by ioanabrooks** [comment]
Fixed in https://github.com/webpack/webpack/releases/tag/v5.83.1
